src/X.Extensions.Logging.Telegram/LogQueueProcessor.cs

```diff
@@ -54,7 +59,10 @@ public void EnqueueMessage(string message)
         // Adding is completed so just log the message
         try
         {
-            _writer.Write(message);
+            foreach (var message in messages)
+            {
+                _writer.Write(message);
```

Comment:
We need some retries mechanism to handle TG bot API rate limits

Reply:
Agree. I will propose changes.

Reply:
https://github.com/dncuug/X.Extensions.Logging.Telegram/pull/25